can/interfaces/pcan/pcan.py

```diff
@@ -82,7 +86,7 @@ def __init__(self, channel, *args, **kwargs):
         else:
             self.channel_info = channel
 
-        bitrate = kwargs.get('bitrate', 500000)
+        bitrate = kwargs.get('bitrate', can.rc.get('bitrate', 500000))
```

Comment:
This should be solved on a higher level instead so that it works for all interfaces.

Reply:
You are right. I removed it.